### ISSUE TITLE
Settings UI - value description is not being read by Screen Reader

### ISSUE THREAD

**Issue by BhanuPrakashGit** [microsoft/vscode#180115]
**GitHub Tags:**
#A11yMAS; #A11yTCS; #A11ySev2; #Device; #Visual Studio Code Client; #BM-VisualStudioCodeClient-MAC-Oct2022; WCAG1.3.1; #Voiceover; #Screenreader; #Info and Relationships;

### Environment and OS details:
Visual Studio Code Insiders
Screenreader: Voiceover

### Repro Steps:

1. Open Visual studio code editor
2. Open the command Pallet and type "setting".
3. Using down-arrow key select user setting.
4. Type the word "Auto " and select one of the suggestion given by the vs code editor.
5. Type the "Tab size" and select the option from the auto suggestion.
6. Verify that Voiceover is not announcing the message which appears when focus comes on "Auto".

### Actual Result:
Voiceover is not announcing the message (Remove adjacent quotes ....) which appears when focus comes on "Auto".

### Expected Result:
Voiceover should announce the message (Remove adjacent quotes ....) which appears when focus comes on "Auto".

### Attachment:
![197955500-494e93b0-438e-4e22-9366-8c232174be82](https://user-images.githubusercontent.com/104351566/232458933-c6db941a-bbf3-4328-a039-f9426229ac47.gif)
![198995371-aa7df2ce-cd65-468a-b9dc-6a2c5fd01826](https://user-images.githubusercontent.com/104351566/232458958-f16c8575-d6e0-4ee2-8726-415f9d8c0184.gif)

**Comment by rzhao271** [comment]
@isidorn I noticed that on macOS on Chrome-based browsers, VoiceOver doesn't read the aria-label of a div if the role of the div is set to "option". I'm wondering whether we could remove the "option" role of the div elements for this specific case, i.e. custom selectboxes on macOS?

**Comment by BhanuPrakashGit** [comment]
Verified the bug in the environment:
Version: 1.45.0-insider (Universal)
Commit: b77ba86a99f525e81954414f9d717d90ccd2a1ca
Date: 2023-05-22T05:32:52.588Z
VS Code: 1.70.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin arm64 22.4.0
Bug is not fixed and still repro'ing.
Could you please reopen this issue.
[180115_Reopened.zip](https://github.com/microsoft/vscode/files/11574874/180115_Reopened.zip)

**Comment by rzhao271** [comment]
@BhanuPrakashGit please try again with today's Insiders.

**Comment by BhanuPrakashGit** [comment]
@rzhao271 
Tried to download latest insiders but still we get the same version.
Version: 1.45.0-insider (Universal)
Commit: 2717a9b16d47cc4c4d5005068bc5fb3ee3de27c4
Date: 2023-05-26T05:31:07.551Z
VS Code: 1.70.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin arm64 22.4.0

Issue is still repro'ing. PFA for reference
Could you please share the downloading link for the latest insiders where fixes are available for this issue.
[180115_NotFixed.zip](https://github.com/microsoft/vscode/files/11588787/180115_NotFixed.zip)

**Comment by fsteffi** [comment]
#regressed:05-29-23;

**Comment by rzhao271** [comment]
Hi @BhanuPrakashGit, that doesn't look like the right version of Insiders to me. Please try again after downloading the latest Insiders from https://code.visualstudio.com/Insiders/#osx.

**Comment by BhanuPrakashGit** [comment]
**GithubTags:**#Regressed:05-31-23;#Closed;
Verified the bug in the environment: 
Version: 1.79.0-insider (Universal)
Commit: fa8eefda18bf099d82b58453f8fe5d3010196fa0
Date: 2023-05-31T03:12:18.068Z
Electron: 22.5.4
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Darwin arm64 22.5.0

Bug is fixed and does not repro. Hence closing the bug.
PFA for reference.

[180115_closed.zip](https://github.com/microsoft/vscode/files/11609791/180115_closed.zip)
